aidial_adapter_bedrock/llm/converse/input.py

```diff
@@ -146,12 +170,31 @@ def tool_result_to_content_part(
         }
 
 
-def to_converse_image_type(type: str) -> str:
-    if type == "image/png":
-        return "png"
-    if type == "image/jpeg":
-        return "jpeg"
-    raise RuntimeServerError(f"Unsupported image type: {type}")
+def to_converse_multi_modal_part(
+    resource: Resource,
+) -> ConverseImagePart | ConverseDocumentPart:
+    if resource.type in IMAGE_MIME_TO_CONVERSE_TYPE:
+        return ConverseImagePart(
+            image=ConverseImagePartConfig(
+                format=IMAGE_MIME_TO_CONVERSE_TYPE[resource.type],
+                source={"bytes": resource.data},
+            )
+        )
+    elif resource.type in DOCUMENT_MIME_TO_CONVERSE_TYPE:
+        return ConverseDocumentPart(
+            document=ConverseDocumentPartConfig(
+                format=DOCUMENT_MIME_TO_CONVERSE_TYPE[resource.type],
+                # Passing generic name, since provider explicitly stated,
+                # that this field is vulnerable for prompt injection, because the model might
+                # inadvertently interpret it as instructions.
+                name=str(uuid.uuid4()),
+                source={"bytes": resource.data},
+            )
+        )
+    else:
+        raise RuntimeServerError(
```

Comment:
Use `UserError` since the Chat user could do something with it e.g. remove the offending attachment.

We already have a similar logic in `DialResource`. Try to reuse it.

https://github.com/epam/ai-dial-adapter-bedrock/blob/development/aidial_adapter_bedrock/dial_api/resource.py#L26-L33

Reply:
Fixed

Reply:
Now in UI it will look like this:

![Screenshot 2024-12-19 at 14 28 28](https://github.com/user-attachments/assets/6590dc87-825d-4710-9857-0d5bfed8e428)

![Screenshot 2024-12-19 at 14 31 07](https://github.com/user-attachments/assets/0f735d25-e947-4ab7-a092-008d87f5307a)

Reply:
The only caveat here:

for `unsupported attachment type` I use MIME type, because there could be some unknown type like `application/some-custom-type`

But for `Supported ... types` I use Converse types, because they're more laconic and user-readable then:
```
Supported document types: application/vnd.openxmlformats-officedocument.wordprocessingml.document, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, ...
```